### DISCUSSION TITLE
Blueprint section field prefixes

### DISCUSSION THREAD

**Discussion by aklos** [statamic/cms#7694] [Q&A] [answered]
When creating sections in a blueprint, they each have a handle but when I add fields to them those fields don't get any prefix/nesting. 

That means I can't have something like Main->Title, SectionB->Title, and SectionC->Title. Statamic will complain that "title" is already being used elsewhere. Right now I have to manually prefix every single field, and if I forget or mess it up the template breaks.

Is there a way to set section field prefixing, or nesting of fields?

**Comment by duncanmcclean** [comment, marked as answer]
Not for blueprint sections, no. You can do it if you're pulling in fields from a fieldset though.

**Comment by martinsnajdr** [comment]
Logically, sections should be able to prefix fields inside, i.e. they should work similarly to group fields. Or at least it would be good if group fields could hide their UI completely, so that there is no duplicate UI when a group field is used to "prefix" fields inside a section.

**Reply by duncanmcclean** [reply]
Regarding hiding the UI for group fields, feel free to add your 👍  to this feature request: https://github.com/statamic/ideas/issues/1137
